experimental/packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -378,10 +380,10 @@ export class FetchInstrumentation extends InstrumentationBase<
               plugin._addHeaders(options, url);
               plugin._tasksCount++;
               return original
-                .apply(this, options instanceof Request ? [options] : [url, options])
+                .apply(self, options instanceof Request ? [options] : [url, options])
```

Comment:
Is this change required or just a style thing?

Reply:
TypeScript is complaining about arrow functions captured a `this` typed as `globalThis`. I added a comment for explaining this.